lib/activerecord-import/import.rb

```diff
@@ -375,15 +411,57 @@ def import_without_validations_or_callbacks( column_names, array_of_attributes,
         post_sql_statements = connection.post_sql_statements( quoted_table_name, options )
 
         # perform the inserts
-        number_inserted = connection.insert_many( [ insert_sql, post_sql_statements ].flatten,
+        (number_inserted,ids) = connection.insert_many( [ insert_sql, post_sql_statements ].flatten,
                                                   values_sql,
                                                   "#{self.class.name} Create Many Without Validations Or Callbacks" )
       end
-      number_inserted
+      [number_inserted, ids]
     end
 
     private
 
+    def set_ids_and_mark_clean(models, import_result)
+      unless models.nil?
+        import_result.ids.each_with_index do |id, index|
+          models[index].id = id.to_i
+          models[index].instance_variable_get(:@changed_attributes).clear # mark the model as saved
+        end
+      end
+    end
+
+    def import_sub_objects(models, options)
```

Comment:
In the code, the comments, tests, and any other references. Just picking one name, one version of it, and then using it consistently. You guys did all the hard work already so if it seems that I'm nit-picking, know that i am just playing the role of editor|reviewe.